sdk/storage/azure-storage-files-shares/test/ut/share_file_client_test.cpp

```diff
@@ -1692,6 +1692,51 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_TRUE(client1.GetUrl().find("snapshot=" + timestamp2) == std::string::npos);
   }
 
+  TEST_F(FileShareFileClientTest, Audience_PLAYBACKONLY_)
```

Comment:
why is this plaback-only?

Reply:
Because it's file oauth. Currently, the pipeline account can't support file oauth.